wherehows-dao/src/main/java/wherehows/dao/DatasetColumnRowMapper.java

```diff
@@ -56,21 +56,22 @@ public DatasetColumn mapRow(ResultSet rs, int rowNum) throws SQLException {
         String strDistributed = rs.getString(DISTRIBUTED_COLUMN);
         boolean distributed = isNotBlank(strDistributed) && strDistributed.equalsIgnoreCase("y");
         DatasetColumn datasetColumn = new DatasetColumn();
-        datasetColumn.id = id;
-        datasetColumn.sortID = sortID;
-        datasetColumn.parentSortID = parentSortID;
-        datasetColumn.fieldName = fieldName;
-        datasetColumn.fullFieldPath = fullPath;
-        datasetColumn.dataType = dataType;
-        datasetColumn.distributed = distributed;
-        datasetColumn.partitioned = partitioned;
-        datasetColumn.indexed = indexed;
-        datasetColumn.nullable = nullable;
-        datasetColumn.comment = comment;
-        datasetColumn.commentCount = commentCount;
-        datasetColumn.treeGridClass = "treegrid-" + Integer.toString(datasetColumn.sortID);
-        if (datasetColumn.parentSortID != 0) {
-            datasetColumn.treeGridClass += " treegrid-parent-" + Integer.toString(datasetColumn.parentSortID);
+        datasetColumn.setId(id);
```

Comment:
One second thoughts, if going down the JPA ORM route is not worth the effort, we can potentially look at Apache commons DbUtils http://commons.apache.org/proper/commons-dbutils/index.html to improve DAO. It's a lightweight extension to JDBC and might be able to help reducing most of our (bad) DAO code.